zmon_agent/main.py

```diff
@@ -97,57 +118,71 @@ def sync(infrastructure_account, region, entity_service, verify, dry_run, interv
 
     while True:
         try:
-            zmon_client = get_clients(entity_service, verify=verify)
+            sync_span = opentracing.tracer.start_span(operation_name='zmon-agent-sync')
+
+            with sync_span:
+                zmon_client = get_clients(entity_service, verify=verify)
```

Comment:
The span is missing the account, region, team? cluster_id tags

Reply:
Added to sync_span